cli/commandhandler.h

```diff
@@ -63,11 +65,11 @@ bool init()
 void handleLoadPemFile(const std::map<uint32_t, std::string> &userInputs)
 {
     ih::wrapper::PemHandlerInputWrapper const pemInputWrapper(userInputs);
-    ih::PemInputHandler pemHandler(pemInputWrapper);
+    ih::PemFileReader pemHandler(pemInputWrapper.getPemFilePath());
 
     if (pemHandler.isFileValid())
     {
-        pemHandler.printFileContent();
+        std::cerr << "Bech32 address: " << pemHandler.getAddress().getBech32Address() << "\n";
```

Comment:
Good point, I also had in mind to use a logger some time in the future.
About `reportError` you are right, it would be more consistent to just throw errors instead of having mixed: exceptions throw AND error logging. 
For now, we've only refactored SDK for error throwing. We should make it consistent for CLI as well in a future PR.